sites/watertechonline.com/config/gam.js

```diff
@@ -1,6 +1,14 @@
 const configureGAM = require('@endeavor-business-media/package-shared/config/gam');
 
-const config = configureGAM({ basePath: 'WTO' });
+const config = configureGAM({
+  basePath: 'WTO',
+  lazyLoad: {
```

Comment:
The lazyload config needs to be directly set on the `config` object after its constructed - not passed to the `configureGAM` function. For an example, see the GAM config in Officer.